tests/apps/testapp_content/urls.py

```diff
@@ -1,6 +1,4 @@
-"""
-Represents protected content
-"""
+"""Represents protected content."""
```

Comment:
"protected content" is a bit terse and therefore confusing at first. Took me a second to get what you meant.